addons/vehicles/CfgEventHandlers.hpp

```diff
@@ -23,6 +23,19 @@ class Extended_Engine_EventHandlers {
             clientEngine = QUOTE(if (local driver (_this select 0)) then {_this call FUNC(startEngine)};);
         };
     };
+    class AllVehicles {
+        class ACE_EngineState {
```

Comment:
```suggestion
        class ACE_EngineState {
```

Reply:
Can do but then you should change the other classes from speed limiter as well but in a different PR

Reply:
No, that's already released. This isn't. Don't follow bad old examples. :laughing: